content/applications/sales/members.rst

```diff
@@ -0,0 +1,108 @@
+=======
+Members
+=======
+
+The **Members** application is where all operations related to memberships can be configured and
+managed. The **Members** app integrates with the **Sales** and **Accounting** applications to sell
+and invoice memberships directly to customers.
+
+Membership products
+===================
+
+To create a new membership product, navigate to :menuselection:`Members app --> Configuration -->
+Membership Products`, and click :guilabel:`New` to open a blank product record.
+
+On the new product form, enter a name for the product in the field under :guilabel:`Product Name`.
+Use the two blank fields to the right of :guilabel:`Membership Duration` to set a date from which
+membership becomes active. Finally, use the :guilabel:`Membership Fee` field to set the price at
+which the product is sold to customers.
+
+The following optional fields can also be configured for each product:
+
+An internal reference number can be set in the field to the right of :guilabel:`Internal Reference`,
+and a category set using the dropdown to the right of :guilabel:`Product Category`. Set a company
```

Comment:
```suggestion
and a category set using the drop-down menu to the right of :guilabel:`Product Category`. Set a company
```
Drop-down is always hyphenated and followed by either "menu" or "window" -- depending on the circumstance